advanced/Scripts/list.sh

```diff
@@ -241,7 +245,8 @@ for var in "$@"; do
   case "${var}" in
     "-w" | "whitelist"   ) listMain="${whitelist}"; listAlt="${blacklist}";;
     "-b" | "blacklist"   ) listMain="${blacklist}"; listAlt="${whitelist}";;
-    "-wild" | "wildcard" ) listMain="${wildcardlist}";;
+    "-wild" | "wildcard" ) listMain="${regexlist}"; wildcard=true;;
```

Comment:
I'd prefer if these were `--wild` and `--regex`. This is a breaking change, but since this is going for 4.0 I think we can break on it.